configure.ac

```
@@ -2040,6 +2040,20 @@ if test "x$with_kvm_openfiles" = "xyes"; then
   with_libkvm="yes"
 fi
 
+AC_CHECK_HEADERS([libpfctl.h], [], [],
+[
+  #include <sys/queue.h>
+  #include <sys/types.h>
+  #include <netinet/in.h>
+  #include <net/if.h>
+  #include <net/pfvar.h>
+])
+AC_CHECK_LIB([libpfctl], [pfctl_status_counter],
```

I think you need to drop the `lib` prefix: https://www.freshports.org/net/libpfctl/ says the shared object is called `lib/libpfctl.so`, so we need to link using `-lpfctl`.

```suggestion
AC_CHECK_LIB([pfctl], [pfctl_status_counter],
```

---

yep, doing that now